modules/_labs/badge/css/lib/badge.scss

```diff
@@ -0,0 +1,53 @@
+@keyframes wdc-badge-grow {
+  from {
+    transform: scale(0.85);
+  }
+
+  to {
+    transform: scale(1);
+  }
+}
+
+@mixin wdc-badge {
+  // base styles
+  align-items: center;
+  animation: wdc-badge-grow 0.2s ease;
+  border-radius: 999px;
+  box-sizing: border-box;
+  display: inline-flex;
+  font-family: $wdc-font-family;
+  font-size: 11px;
```

Comment:
This is more of a design question, but using 11px here causes it to be visually off center since the browser won't render the font on sub-pixels. Maybe we should consider using 10 or 12?

Reply:
Good call out. I'll talk to Rick about it. Thanks!

Reply:
(or make the badge 1px taller/shorter)